napari/_qt/_tests/test_qt_viewer.py

```diff
@@ -56,9 +46,7 @@ def test_new_labels(qtbot):
     assert np.sum(view.dims._displayed_sliders) == 0
 
     # Add labels with image already present
-    viewer = ViewerModel()
-    view = QtViewer(viewer)
-    qtbot.addWidget(view)
+    view, viewer = viewermodel_factory()
```

Comment:
just curious here: how does the `viewermodel_factory` handle two instantiations in a single function scope?  and actually... what was previously happening?  Did that first `view` variable just get left in limbo after becoming unbound when we created a second viewer with the same variable name... and therefore never get shutdown?

Reply:
It saves both entries and shuts both down.  We save a list of `QtViewer`s, not just one.

And yes, the previous code just clobbered and left the other `QtViewer` hanging.

Reply:
> And yes, the previous code just clobbered and left the other QtViewer hanging.

Thanks for fixing this!